### ISSUE TITLE
QSelect scroll position goes to top when pushing more options

### ISSUE THREAD

**Issue by gcarpi** [quasarframework/quasar#13700]
### What happened?

I'm using the example of [Dynamic Loading Options](https://quasar.dev/vue-components/select#example--dynamic-loading-options) and when I populate the select with more options the scroll is set to the top.

I created an [issue](https://github.com/quasarframework/quasar/issues/13694) reporting a similar problem. In that issue the scroll is sent to the first selected option and I managed to solve this problem using some virtual scroll methods, as you can see below:

```js
async $_onVirtualScroll ({ index, ref }) {
  const lastIndex = this.filteredOptions.length - 1

  if (index === lastIndex && this.$_canFetchOptions()) {
    await this.$_loadMoreOptions()

    this.$nextTick(() => { // This code fixes the issue #13694
      ref.reset()
      ref.refresh(lastIndex)
    })
  }
 }
```

But this code generated a new problem, sometimes when I push new options for the select, the scroll is sent to the top and I can't scroll down anymore because I'm always thrown up.

It's hard to reproduce because it only happens a few times, so I'll leave a video below as a demonstration:

https://user-images.githubusercontent.com/19765074/173937317-9c638f66-c25a-459c-af94-a12e12b7b134.mov

### What did you expect to happen?

I would like to have some explanation of why this is happening and if I can prevent this problem from happening.

### Reproduction URL

https://codepen.io/guilhermecarpi/pen/gOvqwPr?editors=101

### How to reproduce?

I can't reproduce this issue outside of my project

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

### Additional context

Quasar version: 1.19.1

**Comment by DouglasCalora** [comment]
Please, someone help!! 😢

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.7.2 and 1.19.2
